### ISSUE TITLE
Update deps?

### ISSUE THREAD

**Issue by OmgImAlexis** [voxpelli/node-connect-pg-simple#193]
Any chance on the deps of this being updated?

```
  pg                           ^8.2.1   →      ^8.5.1 
  @types/pg                   ^7.14.4   →     ^7.14.8 
  @types/chai                 ^4.2.13   →     ^4.2.14 
  @types/express              ^4.17.7   →    ^4.17.11 
  @types/express-session      ^1.17.0   →     ^1.17.3 
  @types/node               ^10.17.28   →   ^14.14.21 
  @voxpelli/eslint-config      ^5.0.0   →     ^10.0.0 
  eslint                       ^6.8.0   →     ^7.18.0 
  eslint-config-standard      ^14.1.0   →     ^16.0.2 
  eslint-plugin-es             ^3.0.0   →      ^4.1.0 
  eslint-plugin-jsdoc         ^21.0.0   →     ^31.0.7 
  eslint-plugin-mocha          ^6.2.2   →      ^8.0.0 
  eslint-plugin-node          ^11.0.0   →     ^11.1.0 
  eslint-plugin-standard       ^4.0.0   →      ^5.0.0 
  eslint-plugin-unicorn       ^19.0.1   →     ^26.0.1 
  husky                        ^4.3.0   →      ^4.3.8 
  mocha                        ^8.1.3   →      ^8.2.1 
  @types/mocha                 ^8.0.0   →      ^8.2.0 
  pg-promise                  ^10.0.0   →     ^10.9.1 
  proxyquire                   ^2.1.2   →      ^2.1.3 
  sinon                        ^9.2.0   →      ^9.2.3 
  @types/sinon                 ^9.0.8   →     ^9.0.10 
  supertest                    ^5.0.0   →      ^6.1.1 
  typescript                   ^4.0.3   →      ^4.1.3 
  ```

**Comment by voxpelli** [comment]
@OmgImAlexis Yes, there are automated PR:s like #191 and #192 and I tend to go through them when I find any other reason to do a release, sometimes even more often.

Is there any particular dependency you would find a benefit in me updating?

(I do see now that #191 and #192 are both failing their tests, I'll give that a look)

**Comment by voxpelli** [comment]
I pushed a fix for the failing tests to `master` now @OmgImAlexis, I may merge a few of the dependency update PR:s once they have rebased themselves and the CI has run on them 👍

**Comment by voxpelli** [comment]
I released a pre-release / release candidate of `7.0.0-0` now, with updated dependencies. Thanks for pushing me 🙂
